flocker/node/agents/ebs.py

```diff
@@ -20,12 +20,15 @@
 from boto.utils import get_instance_metadata
 from boto.exception import EC2ResponseError
 from twisted.python.filepath import FilePath
-from eliot import Field, MessageType
 
 from .blockdevice import (
     IBlockDeviceAPI, BlockDeviceVolume, UnknownVolume, AlreadyAttachedVolume,
     UnattachedVolume, get_blockdevice_volume
 )
+from ._logging import (
+    AWS_ACTION, BOTO_EC2RESPONSE_ERROR, NO_AVAILABLE_DEVICE,
+    NO_NEW_DEVICE_IN_OS
+)
```

Comment:
i want to consolidate all node/agents/* logging actions in _logging, if this pattern looked ok to you. i took route/_logging.py as an example for this file. If this pattern looks fine, will consolidate ``block device`` and ``cinder`` logs here as a followup.